docker-compose.yml

```
@@ -190,4 +190,5 @@ services:
 
 networks:
   mynetwork:
-    driver: bridge
\ No newline at end of file
+    driver: bridge
+    
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)